docs/content/tutorial/assets/testing-assets.mdx

```diff
@@ -43,25 +84,23 @@ def test_cereal_assets():
     assert result.output_for_node("highest_protein_nabisco_cereal") == "100% Bran"
 ```
 
```

Comment:
It might be helpful to add a snippet here that shows what the file should look like in its entirety at this point. I'm not totally sure how to structure that in `docs_snippets` though - could use some help there.

Reply:
Hmm yeah, the only thing I can think of would be to have a separate file with the same contents minus the marker comments. IMO it could be nice but not critical.